app/com/arpnetworking/metrics/portal/alerts/AlertRepository.java

```diff
@@ -43,9 +44,10 @@
      * Get the <code>Alert</code> by identifier.
      *
      * @param identifier The <code>Alert</code> identifier.
+     * @param organization The organization owning the alert.
      * @return The matching <code>Alert</code> if found or <code>Optional.empty()</code>.
      */
-    Optional<Alert> get(final UUID identifier);
+    Optional<Alert> get(UUID identifier, Organization organization);
```

Comment:
I took a shot at this.  Ie, keeping organization out of the query args and keeping it as arguments on the methods.  Let me know what you think.